### ISSUE TITLE
NPE in Code Action when hovering the incomplete return statement

### ISSUE THREAD

**Issue by dulajdilshan** [ballerina-platform/ballerina-lang#29471]
**Description:**

**Steps to reproduce:**
* vscode-plugin should be enabled debug mode and should listen to the ballerina channel of the output
1) copy the code snippet below
```ballerina
import ballerina/graphql;

service graphql:Service /graphql on new graphql:Listener(9090) {
    resource function get getPerson() returns string{
        return ;
    }
}
```
SS of the code:
![Screenshot from 2021-03-23 16-39-04](https://user-images.githubusercontent.com/25485997/112137563-6a813380-8bf6-11eb-9351-baf0a3c9ce96.png)
2) Hover the mouse on near the incomplete return statement
3) See the following debug trace on ballerina output
```
[Error - 4:29:34 PM] CodeAction 'FixReturnTypeCodeAction' failed! {, error: 'java.lang.NullPointerException'}
java.lang.NullPointerException
	at org.ballerinalang.langserver.codeaction.providers.changetype.FixReturnTypeCodeAction.getFunctionNode(FixReturnTypeCodeAction.java:133)
	at org.ballerinalang.langserver.codeaction.providers.changetype.FixReturnTypeCodeAction.getDiagBasedCodeActions(FixReturnTypeCodeAction.java:75)
	at org.ballerinalang.langserver.codeaction.CodeActionRouter.lambda$getAvailableCodeActions$2(CodeActionRouter.java:96)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at org.ballerinalang.langserver.codeaction.CodeActionRouter.lambda$getAvailableCodeActions$3(CodeActionRouter.java:94)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1655)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
	at org.ballerinalang.langserver.codeaction.CodeActionRouter.getAvailableCodeActions(CodeActionRouter.java:92)
	at org.ballerinalang.langserver.codeaction.BallerinaCodeActionExtension.execute(BallerinaCodeActionExtension.java:46)
	at org.ballerinalang.langserver.codeaction.BallerinaCodeActionExtension.execute(BallerinaCodeActionExtension.java:34)
	at org.ballerinalang.langserver.LangExtensionDelegator.codeActions(LangExtensionDelegator.java:133)
	at org.ballerinalang.langserver.BallerinaTextDocumentService.lambda$codeAction$8(BallerinaTextDocumentService.java:335)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1692)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
```
**Affected Versions:**
Swan Lake Alpha 3

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by nadeeshaan** [comment]
This issue has been fixed and cannot reproduce in master. Hence closing the issue

**Comment by IMS94** [comment]
This is fixed by #29337 and will be available in the next release.
